rainier-core/src/main/scala/com/stripe/rainier/core/SBC.scala

```diff
@@ -19,6 +19,15 @@ final case class SBC[T, L <: Distribution[T]](priors: Seq[Continuous],
 
   val priorGenerator = Generator.traverse(priors.map(_.generator))
 
+  def posteriorSamples(nSamples: Int)(implicit rng: RNG): List[T] =
```

Comment:
Just realized that this is only exercising the logDensity of the prior, and the generator of the likelihood. I think we should be trying to exercise both functions for both distributions. I think we could do that with something like `model(1000).sample(nSamples)` here.